scipy/stats/_multivariate.py

```diff
@@ -3850,3 +3851,418 @@ def rvs(self, dim, size=1, random_state=None):
 
 
 unitary_group = unitary_group_gen()
+
+_mhg_doc_default_callparams = """\
+m : array_like
+    The number of different types of objects
+    in the population
+n : array_like
+    The number of samples taken from the population.
+"""
+
+_mhg_doc_callparams_note = \
+"""`m` must be an array of positive integers. Quantiles `x` must
+sum up to the sample size `n`. If the quantile :math:`i` contain
+values out of the range :math:`[0, m_i]` where :math:`m_i` is the
+number of objects of type :math:`i` in the population, appropriate
+values (like zero by `pmf` method) are returned by the called
+methods. If `m` or `n` contain negative values, the result will
+contain `nan`s there.
+"""
+
+_mhg_doc_frozen_callparams = ""
+
+_mhg_doc_frozen_callparams_note = \
+    """See class definition for a detailed description of parameters."""
+
+mhg_docdict_params = {
+    '_doc_default_callparams': _mhg_doc_default_callparams,
+    '_doc_callparams_note': _mhg_doc_callparams_note,
+    '_doc_random_state': _doc_random_state
+}
+
+mhg_docdict_noparams = {
+    '_doc_default_callparams': _mhg_doc_frozen_callparams,
+    '_doc_callparams_note': _mhg_doc_frozen_callparams_note,
+    '_doc_random_state': _doc_random_state
+}
+
+class multivariate_hypergeom_gen(multi_rv_generic):
+    r"""
+    A multivariate hypergeometric random variable.
+
+    Methods
+    -------
+    ``pmf(x, m, n)``
+        Probability mass function.
+    ``logpmf(x, m, n)``
+        Log of the probability mass function.
+    ``rvs(m, n, size=1, random_state=None)``
+        Draw random samples from a multivariate hypergeometric
+        distribution.
+    ``mean(m, n)``
+        Mean of the multivariate hypergeometric distribution.
+    ``var(m, n)``
+        Variance of the multivariate hypergeometric distribution.
+    ``cov(m, n)``
+        Compute the covariance matrix of the multivariate
+        hypergeometric distribution.
+
+    Parameters
+    ----------
+    x : array_like
+        Quantiles, with the last axis of `x` denoting the components.
+    %(_doc_default_callparams)s
+    %(_doc_random_state)s
+
+    Notes
+    -----
+    %(_doc_callparams_note)s
+
+    Alternatively, the object may be called (as a function) to fix the `m` and
+    `n` parameters, returning a "frozen" multivariate hypergeometric
+    random variable:
```

Comment:
This looks resolved.

In any case, I would argue that it could be either. This text introduces/announces the part to follow (the code). That's a good use of a colon.

Reply:
I'm ok with leaving as it, marking resolved.